### ISSUE TITLE
parseQuery should get a string as first argument

### ISSUE THREAD

**Issue by kouhin** [bhovhannes/svg-url-loader#51]
webpack 2.0

```
Module build failed: Error: parseQuery should get a string as first argument
```

@bhovhannes This bug is already fixed by loader-utils v0.2.16, but when will the new version of svg-url-loader be released?

**Comment by karmajunkie** [comment]
+1?

**Comment by bhovhannes** [comment]
I've just released version 1.1.1.
The issue should be fixed.
